proto/raze/remote/BUILD.base64-0.9.3.bazel

```diff
@@ -7,7 +7,7 @@ DO NOT EDIT! Replaced on runs of cargo-raze
 
 # buildifier: disable=load
 load(
-    "@io_bazel_rules_rust//rust:rust.bzl",
+    "@rules_rust//rust:rust.bzl",
```

Comment:
This BUILD file (as with all the others generated via `cargo-raze` treats `rules_rust` as an external repository. The `rust_rules_workspace_name` setting can be used to control the name but in order to really support multiple names there would need to be multiple versions of these files that have the new name. Again, the blocker here is the inability to detect the workspace name dynamically.